### DISCUSSION TITLE
Memory leak problem

### DISCUSSION THREAD

**Discussion by cycaoyang** [godka/Pensieve-PPO#28] [Q&A] [answered]
Hello, I am a beginner of reinforcement learning. When I ran your code, I noticed that the server's memory utilization skyrocketed, causing the final workout to stop midway through. So I have a question: Is there a memory leak in your code?

**Comment by godka** [comment, marked as answer]
Hi Caoyang,

We started reproducing the problem you reported. However, we didn't observe a memory leak -- the memory utilization is kept around 380MB, detailed are shown as follows:

![reward](https://user-images.githubusercontent.com/13187954/232328157-4c7806a4-85e3-4b70-ad60-7049f4afbc00.png)

![memory](https://user-images.githubusercontent.com/13187954/232328167-1c937995-432d-421a-8256-569178db123e.png)

Here are the raw logs, including memory and tensorboard logs

[events.out.tfevents.1681531853.zip](https://github.com/godka/Pensieve-PPO/files/11242912/events.out.tfevents.1681531853.zip)

Feel free to let us know if you have any questions.

Best,
Tianchi
